openweather-based-bidding/main.gs

```diff
@@ -26,97 +26,66 @@ const sheetsApi = new SheetsApi(configSpreadsheetId);
  */
 function monitorWeatherAndSyncWithDV360() {
   Logger.log('[START] monitorLineItemChangesAndSyncWithDV360');
+
   // Get items from Sheet
-  const data = sheetsApi.get(configSpreadsheetName);
-  const rows = data['values'];
+  const rows = sheetsApi.get(configSpreadsheetName);
+
+  // Process sheet headers
+  const apiHeaders = Utils.getApiHeaders(rows[0]);
```

Comment:
This confused me quite a bit before I realized that `setHeaders()` in the next line does not process the `apiHeaders`^^
Could we move this down to where `apiHeaders` is actually used?

Reply:
I'm did a refactoring and moved the header functionality to the `config` module. Please let me know if this is more readable
```
  config.setHeaders(rows[0]);
  const apiHeaders = config.getApiHeaders();
```